src/packages/paths.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+import yaml
+
+
+class AllPaths:
+    def __init__(self, config) -> None:
+        self.BASE_DIR = Path(__file__).resolve().parent.parent.parent
+
+        self.config = config
+        self.cremerf_catalog = f'/Volumes/{self.config["catalog_name"]}/{self.config["schema_name"]}/'
+        self.data_volume = f"{self.cremerf_catalog}" + "data/"
+
+
+# Load configuration
+with open("/Users/fcremer29/focus/marvelmlops/project-config.yml", "r") as file:
```

Comment:
This path would not work if you want to run on Databricks.
You can define 2 paths, one local one for Databricks, for example : you upload your project config file to volumes and read from there. Depending on the environment, you use the correct path.

Reply:
There are several ways. What I usually do is creating an environment variable, e.g. ENVIRONMENT = local vs databricks

In your code, you add a if else:

if ENVIRONMENT == local:
   path
else
   path
   
 Something like this. And you only change the environment variable.

Reply:
thank you basak, you mean path from line 11 (self.cremerf_catalog*) or the hardcoded path from 16? (with open("")). 

*self.cremerf_catalog output -> '/Volumes/mlops_students/cremerfederico29/'

because if you mean path from line 11 do you have any idea on how my code could "understand when" is running on each environment (you mentioned local vs databricks)? Then it should be just a control flow to use the correct path, but I don't know how to identify if it's local or databricks